xonsh/xoreutils/echo.py

```diff
@@ -0,0 +1,27 @@
+"""Impelements a simple echo command for xonsh."""
+
+
+def echo(args, stdin, stdout, stderr):
+    """A simple echo command."""
+    opts = _echo_parse_args(args)
+    if opts is None:
+        return
+    ender = opts['end']
+    args = map(str, args)
+    if opts['escapes']:
+        args = map(lambda x: x.encode().decode('unicode_escape'), args)
+    print(*args, end=ender, file=stdout)
+
+
+def _echo_parse_args(args):
+    out = {'escapes': False, 'end': '\n'}
```

Comment:
I don't think so. I didn't want to change the code too much for no reason, but I am not a huge fan of `arg_handerler()` personally.